src/components/form/dt-users-connection/dt-users-connection.js

```diff
@@ -43,6 +43,17 @@ export class DtUsersConnection extends DtTags {
     ];
   }
 
+  static get properties() {
+    return {
+      ...super.properties,
+      /**
+       * The name attribute used to identify an input within a form.
+       * This will be submitted with the form as the field's key.
```

Comment:
Can you update this description to correctly describe the `single` property? It gets used in automatically generating documentation.